dojo/views.py

```diff
@@ -13,9 +13,10 @@
 from dojo.models import Engagement, Test, Finding, Endpoint, Product, FileUpload
 from dojo.filters import LogEntryFilter
 from dojo.forms import ManageFileFormSet
-from dojo.utils import get_page_items, Product_Tab, get_system_setting
+from dojo.utils import get_page_items, Product_Tab
 from dojo.authorization.authorization import user_has_permission, user_has_permission_or_403, user_has_configuration_permission_or_403
 from dojo.authorization.roles_permissions import Permissions
+from dojo.settings.settings import ENABLE_AUDITLOG
```

Comment:
It would be better to use the registered version in `django.conf.settings` rather than importing from the file directly

Reply:
Done